localenv/mock-account-servicing-entity/app/entry.server.tsx

```diff
@@ -39,20 +42,22 @@ if (!global.__seeded) {
     )
   }
 
-  callWithRetry(async () => {
-    console.log('setting up from seed...')
-    return setupFromSeed(CONFIG, apolloClient, mockAccounts, {
-      logLevel: 'debug',
-      pinoPretty: true
-    })
-  })
-    .then(() => {
+  callWithRetry(
+    async (): Promise<{ tenantId: string; apiSecret: string } | undefined> => {
+      console.log('setting up from seed...')
+      return setupFromSeed(CONFIG, generateApolloClient, mockAccounts, {
+        logLevel: 'debug',
+        pinoPretty: true
+      })
+    }
+  )
+    .then((seedResult: { tenantId: string; apiSecret: string } | undefined) => {
       global.__seeded = true
       setTimeout(() => {
         const url = new URL(`http://localhost:${process.env.FRONTEND_PORT}/`)
         const params = new URLSearchParams({
-          tenantId,
-          apiSecret
+          tenantId: seedResult?.tenantId ?? tenantId,
```

Comment:
small thing, but the log always still shows: `Use this URL to access the frontend with operator tenant credentials`

think we could use `seedResult.tenantId` to omit the `operator` in that message, or just make the message more generic.